src/main/java/org/apache/commons/dbcp2/PoolingConnection.java

```diff
@@ -122,7 +121,6 @@ public synchronized void close() throws SQLException {
             }
         } finally {
             try {
-                @SuppressWarnings("resource")

```

Comment:
Do not remove `SuppressWarnings`. There are there for a reason: IDEs can be configured to show warnings, this documents that a warning can be ignored and shows when Closeable and AutoCloseable resources are intentionally left open.

Reply:
I think modern versions of IntelliJ suggest to remove this warnings. Thanks for the context though. With that being said, there is no useful changes in the PR to merit a review. I am gonna close this. Sorry for wasting your time.